packages/babel-types/scripts/generators/typescript.js

```diff
@@ -106,7 +106,7 @@ for (const type in t.NODE_FIELDS) {
 for (let i = 0; i < t.TYPES.length; i++) {
   let decl = `export function is${
     t.TYPES[i]
-  }(node: object, opts?: object | null): `;
+  }(node: any, opts?: object | null): `;
```

Comment:
I think that it should be `object | null | undefined`, so that it will still warn for things like `t.isIdentifier(node.type)`.

Reply:
I guess I'd be OK with this. Technically `t.isIdentifier("foo")` works just fine and returns `false`, but actually passing something like this is definitely a code smell...

I'm not worried about length, so I'll update to `object | null | undefined`. I'm not very familiar with Flow but I'll give that a shot and let you double check it.